BridgeAppSDK/SBAUserProfileController.swift

```diff
@@ -37,7 +37,40 @@ import Foundation
  Protocols for sharing functionality between different classes that do not share inheritance.
  This set of protocols are used to handle account access.
  */
-public protocol SBAUserProfileController: class, SBAAccountController, SBAResearchKitResultConverter {
+public protocol SBAUserProfileController: class, SBAAccountController, SBAResearchKitResultConverter, SBANameDataSource {
+}
+
+extension SBAUserProfileController {
+    
+    func updateUserProfileInfo() {
+
+        if let name = self.name {
```

Comment:
should this be givenName? or are givenName and fullName still conflated into the name field in the profile?